--- dosfstools-2.2/mkdosfs/mkdosfs.c.2.88	Wed Sep 29 12:58:40 1999
+++ dosfstools-2.2/mkdosfs/mkdosfs.c	Wed Sep 29 12:59:13 1999
@@ -627,6 +627,10 @@ establish_params (int device_num,int siz
 	      param.sect = 18;
 	      param.head = 2;
 	      break;
+	    case 5760:
+	      param.sect = 36;
+	      param.head = 2;
+	      break;
 	    default:
 	      /* fake values */
 	      param.sect = 32;
